### ISSUE TITLE
[idea] Node.empty()

### ISSUE THREAD

**Issue by vrugtehagel** [whatwg/dom#955]
A common thing to do when manipulating DOM elements is "emptying" an element, i.e. removing all its child nodes. There are currently a few solutions to this; let me list the most obvious ones.
```javascript
element.innerHTML = '';
```
This solution is semi-readable, but invokes the DOM parser (unless specifically optimized for by the browser).
```javascript
element.textContent = '';
```
Similar to the previous one, but less readable (as the intended operation doesn't really have anything to do with the text content specifically). This is supposedly more performant than `innerHTML`.
```javascript
while(element.firstChild){
    element.removeChild(element.lastChild);
}
```
there are various variations on this one, none of them marginally better than another. This solution works, but requires three lines of code for a simple operation that, in my opinion, should be standardized.

So, I propose adding a method like `.empty()` to the Node interface. What are your thoughts on this?

**Comment by domenic** [comment]
A dupe of https://github.com/whatwg/dom/issues/478, although the original post there seems to have been edited to reflect the eventual conclusion of the thread; it started very much like this.

TLDR: use `el.textContent = ""` (which is optimized in browsers) or `el.replaceChildren()`.

**Comment by vrugtehagel** [comment]
I'm not sure I'm content with how that reads, but I suppose I could always implement a more appropriately named method myself. Thanks for pointing me to that post.
